README.md

```diff
@@ -101,6 +105,20 @@ In a separate terminal, contracts can be deployed using
 ```
 One of the contracts that gets deployed locally is a mock contract that represents the $BOSON token. The mock exists for unit testing purposes and so that those who want to develop against the protocol locally don't have to point to a testnet deployment of the $BOSON token.
 
+---
+### Run (Windows)
+On Windows, the 'go' script is not supported. The standard Truffle and Ganache commands need to be used.
+
+Ganache can be started up manually in a terminal using the command:
+```shell
```

Comment:
I don't think that here it should not say `shell`, as `shell` is not being used, but the `cmd` or `Powershell`, as it uses the Windows paths. It is misleading, as, for instance, `git-bash` can be used on Windows environment as well, which uses the UNIX path spec.

Reply:
'shell' in the README file is a markdown formatting instruction and does not appear in the rendered file: